docs/admin/cluster-management.md

```diff
@@ -66,37 +66,52 @@ Instance Group will take care of putting appropriate image on new machines and s
 In other environments you may need to configure the machine yourself and tell the Kubelet on which machine API server is running.
 
 
-### Horizontal auto-scaling of nodes (GCE)
+### Cluster autoscaling
 
-If you are using GCE, you can configure your cluster so that the number of nodes will be automatically scaled based on:
+If you are using GCE or GKE, you can configure your cluster so that it is automatically rescaled based on
+pod needs.
 
- * CPU and memory utilization.
- * Amount of of CPU and memory requested by the pods (called also reservation).
+As desribed in [Compute Resource](../user-guide/compute-resources.md), user can declare how much
+CPU and memory a pod needs to run. This amount is called request and is used by Kubernetes scheduler (along with ports
+and labels, if provided) to find a place to start the pod. If there is no node that has enough free capacity or
+doesn't match other pod requirements then the pod has to wait, possibly quite long,
+until some pods are terminated. Or until a new node is added. 
```

Comment:
How about:

As described in [Compute Resource](/docs/user-guide/compute-resources/), users can reserve how much CPU and memory is allocated to pods. This information is used by the Kubernetes scheduler to find a place to run the pod. If there is no node that has enough free capacity (or doesn't match other pod requirements) then the pod has to wait until some pods are terminated or a new node is added.